var/spack/repos/builtin/packages/libsonata/package.py

```diff
@@ -36,6 +37,7 @@ class Libsonata(CMakePackage):
     def cmake_args(self):
         result = [
             '-DEXTLIB_FROM_SUBMODULES=OFF',
+            '-DSONATA_TESTS=OFF',
```

Comment:
I saw this https://github.com/BlueBrain/libsonata/compare/v0.1.3...v0.1.4#diff-2eeaed663bd0d25b7e608891384b7298
Not very sure if I have to add it here though

Reply:
hm, I thought you need to adjust `py-libsonata/package.py`. Check other PRs of bluepysnap and see what files it handled there `libsonata/package.py`, `py-libsonata/package.py` or may be both.